File: app/src/main/java/com/jorbital/gymstat/utils/DayOfWeekRealm.java
package com.jorbital.gymstat.utils;

import org.threeten.bp.DayOfWeek;

import io.realm.RealmObject;

public class DayOfWeekRealm extends RealmObject
{
    private String dayOfWeek;

    public void setEnum(DayOfWeek val)
    {
        this.dayOfWeek = val.toString();
    }

    public DayOfWeek getEnum()
    {
        return DayOfWeek.valueOf(dayOfWeek);
    }
}

File: app/src/main/java/com/jorbital/gymstat/utils/DayOfWeekRealm.kt
package com.jorbital.gymstat.utils

import io.realm.RealmObject
import org.threeten.bp.DayOfWeek

open class DayOfWeekRealm : RealmObject() {
    private var dayOfWeek: String = ""

    var enum: DayOfWeek
        get() = DayOfWeek.valueOf(dayOfWeek)
        set(`val`) {
            this.dayOfWeek = `val`.toString()
        }
}
